hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/condition/AvailabilityCondition.java

```diff
@@ -16,8 +16,11 @@
  */
 package org.hawkular.alerts.api.model.condition;
 
+import static org.hawkular.alerts.api.model.trigger.Trigger.Mode.FIRE;
+
 import org.hawkular.alerts.api.log.MsgLogger;
```

Comment:
Another thing that I have on my todo list is to review the logger dependency in the API.
In the conditions we have the match() logic and we have to log a possible error if operator is not correct.
Thinking this model can be shared, I think this is a reason to have a more DTO oriented approach.
We can have all match() methods as a part of the engine implementation, then we can also simplify the model and get rid of the logging dependencies for the hawkular-alerts-api artifact what is good.